Resources/Changelog/GabyChangelog.yml

```
@@ -1,77 +1,238 @@
+- author: Dreykor(arguemeu)
+  time: '2025-03-08T12:01:00.0000000+00:00'
+  changes:
+    - type: Fix
+      message: readiciona whitelist para hamburguers, assim evitando o meta de insta kill, e evitando que itens aleatorios sejam botados no hamburguer, algo sem muito sentido em MRP.
+- author: Dreykor (arguemeu)
+  changes:
+    - type: Add
+      message: Traduz o texto de eventos ERT, do novo alerta Omicron, e altera o audio do alerta epsilon para sua versão completa.
+  id: 31
+  time: '2025-03-17T21:30:00.0000000+00:00'
 Name: GabyStation
```

YAMLlint (1.35.1)

[error] duplication of key "Name" in mapping

(key-duplicates)